flake.nix

```
@@ -102,7 +132,11 @@
         packages.default = self'.packages.rio;
         devShells.default = self'.devShells.msrv;
 
-        packages.rio = rustPackage "rio";
+        apps.default = {
+          type = "app";
+          program = self'.packages.rio;
```

```suggestion
          program = self'.packages.default;
```
This would be better to point to rather than `packages.rio` for just the sake of keeping it logical and clean.